x-pack/docs/en/security/securing-communications/node-certificates.asciidoc

```diff
@@ -75,12 +75,27 @@ signing requests (CSR) for the nodes in your cluster. For more information, see
 <<certutil>>.
 --
 
-. Copy the node certificate to the appropriate locations.
+. Optional: Generate additional certificates specifically for encrypting HTTP
+client communications. 
 +
 --
-Copy the applicable `.p12` file into a directory within the {es} configuration
-directory on each node. For example, `/home/es/config/certs`. There is no need
-to copy the CA file to this directory.
+For example, use the `elasticsearch-certutil http` command:
+
+[source,shell]
+----------------------------------------------------------
+bin/elasticsearch-certutil http
+----------------------------------------------------------
+
+This command guides you through the process of generating the appropriate
+certificates for use in {es} and {kib}. If you created a CA for your cluster,
+you can re-use it by supplying its location when prompted.
+--
+
+. Copy the node certificates to the appropriate locations.
++
+--
+Copy the applicable `.p12` files into the {es} configuration directory on each
```

Comment:
```suggestion
Copy the applicable files into the {es} configuration directory on each
```

It will most probably be one or two `p12` files but not _always_ . ( They might get  a CSR to give to their CA, for instance ). The output of the `http` subcommand is pretty detailed so I don't think we can be a little generic here